### ISSUE TITLE
question about the paper in ablation studies

### ISSUE THREAD

**Issue by yanan1116** [Koorye/Inspire#6]
Hi authors, 

i have a question about the performance of the +inspire model

<img width="972" height="456" alt="Image" src="https://github.com/user-attachments/assets/43a8bb59-f6ca-4e7e-b912-ee368b4352ec" />
as you can see in the table 1, you use `miniVLA-VQ` and `π0-FAST` as the primary baseline methods. 
and according to the experimental setup part here:

<img width="966" height="282" alt="Image" src="https://github.com/user-attachments/assets/0f71281e-2b3b-42b9-b150-b4c4229d107d" />

for the two models, 
 - are they off-the-shelf and directly borrowed from the original paper ? 
 - have you train / sft them on your self-made `Spatial Reasoning VQA` dataset which contains spatial reasoning VQA as the 'bridge language' between obs and actions, **but without the 'bridge VQA' part** ? 
 

since the 'bridge' VQA part is the core the paper and you futher train the baseline models on VQA-enhanced labelled dataset with the autoregressive loss. I am not sure the gains are from additional training or from additional training with VQA. 
(the VQA part is via Automated Rule-based Object Direction Labeling as you stated in the paper).

I have not found any words talking about this ablation.

and since the your method needs training, may I know why it is a `plug-and-play` manner ?

<img width="1005" height="210" alt="Image" src="https://github.com/user-attachments/assets/dbce72f8-aa37-449f-969d-3c6bf3527256" />

thanks a lot.

**Comment by yanan1116** [comment]
i suppose in the script of `vla_scripts/train/train_inspire_libero90.sh`, you train the model `prism-qwen25-dinosiglip-224px+0_5b+mx-libero-90` with VQA dataset, which is from object-robot position relationship annotation.

but what is the training set for `vla_scripts/train/train_baseline_libero90.sh` ? does it come from https://github.com/Koorye/Inspire/blob/main/DATASET.md  somewhere in `tensorflow_datasets` folder ?

Could you elaborate ?

**Comment by Koorye** [comment]
Thank you for your attention to our work.  

1. In Table 1, the "Seen" results for models marked with * are from the official report at https://ai.stanford.edu/blog/minivla/, while the "Unseen" results were obtained by testing on officially released models. For models marked with †/‡, respectively, they were pre-trained or fine-tuned on LIBERO-90 to obtain these results.

2. We did not introduce any additional data; all training (including fine-tuning) data is sourced from LIBERO-90. We generated object-robot position relationship annotations on LIBERO-90, where the model predicts these relationship annotations in a self-regressive manner and then proceeds to predict actions. —all to be done in a single training phase without an additional "relationship prediction" stage.

3. The `plug-and-play` approach mentioned in this paper refers to our Inspire module being able to be plugged into different VLA architectures while simultaneously improving their performance.
